Lib/test/test_asyncio/test_events.py

```diff
@@ -2912,14 +2901,10 @@ def test_get_event_loop_returns_running_loop2(self):
             loop = asyncio.new_event_loop()
             self.addCleanup(loop.close)
 
-            with self.assertWarns(DeprecationWarning) as cm:
-                loop2 = asyncio.get_event_loop()
-            self.addCleanup(loop2.close)
-            self.assertEqual(cm.filename, __file__)
-            asyncio.set_event_loop(None)
             with self.assertRaisesRegex(RuntimeError, 'no current'):
                 asyncio.get_event_loop()
 
+            asyncio.set_event_loop(None)
             with self.assertRaisesRegex(RuntimeError, 'no running'):
                 asyncio.get_running_loop()
```

Comment:
It tests get_running_loop().

Reply:
This is not easy, because suggestion does not work for not edited lines. Just test
```py
            with self.assertRaisesRegex(RuntimeError, 'no current'):
                asyncio.get_event_loop()
```
before and after `asyncio.set_event_loop(None)`. There is no longer need to check `get_running_loop()` and `_get_running_loop()` here.